jbang version --update does not work the second time

```
 jbang version --check
[jbang] There is a new version of jbang available!
[jbang] You have version 0.68.0 and 0.69.1 is the latest.
[jbang] Run 'jbang version --update' to update to the latest version.
0.68.0
❯ which jbang
/Users/max/.jbang/bin/jbang
❯ jbang version --update
[jbang] Downloading and installing jbang...
0.68.0
❯ jbang version --update --verbose
[jbang] jbang version 0.68.0
[jbang] Downloaded file https://github-releases.githubusercontent.com/230011262/eef3a680-8c36-11eb-8d85-071e7c8391c7?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210325%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210325T074456Z&X-Amz-Expires=300&X-Amz-Signature=e1887c39d388609c6bae759cdb0a42faff76e4f801ea2d785c873501f12b0e3a&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=230011262&response-content-disposition=attachment%3B%20filename%3Dversion.txt&response-content-type=application%2Foctet-stream
[jbang] Downloading and installing jbang...
[jbang] Retrieved file from cache https://www.jbang.dev/releases/latest/download/jbang.zip = /Users/max/.jbang/cache/urls/5a954c14968e5a7bad565175cebae4982122dac51e5a4d594e3650593eb17330/jbang.zip
[jbang] Deleting folder /Users/max/.jbang/cache/urls/jbang
[jbang] Deleting file /Users/max/.jbang/bin/jbang.ps1
[jbang] Deleting file /Users/max/.jbang/bin/jbang.jar
[jbang] Deleting file /Users/max/.jbang/bin/jbang
[jbang] Deleting file /Users/max/.jbang/bin/jbang.cmd
Cache: /Users/max/.jbang/cache
Config: /Users/max/.jbang
Repository:/Users/max/.m2/repository
0.68.0
```

jbang version update fetches url from cache instead of forcing fresh one.

jbang cache url makes it happen whats weird though is that it still prints old version after the install...but it works

```
❯ jbang cache clear --url
[jbang] Clearing cache for urls
❯ jbang version --update --verbose
[jbang] jbang version 0.68.0
[jbang] Downloaded file https://github-releases.githubusercontent.com/230011262/eef3a680-8c36-11eb-8d85-071e7c8391c7?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210325%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210325T074825Z&X-Amz-Expires=300&X-Amz-Signature=65c6ac695d44afb015c91db9d247913d6d9f184619e6d5e7b42408d16e03b8e4&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=230011262&response-content-disposition=attachment%3B%20filename%3Dversion.txt&response-content-type=application%2Foctet-stream
[jbang] Downloading and installing jbang...
[jbang] Downloaded file https://github-releases.githubusercontent.com/230011262/ed29e300-8c36-11eb-94d0-b41c5e8cbb39?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210325%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210325T074826Z&X-Amz-Expires=300&X-Amz-Signature=925c92867a482b322a1c1f92a3ef2116b533ba294f29559a00772e6dd073bde9&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=230011262&response-content-disposition=attachment%3B%20filename%3Djbang.zip&response-content-type=application%2Foctet-stream
[jbang] Deleting file /Users/max/.jbang/bin/jbang.ps1
[jbang] Deleting file /Users/max/.jbang/bin/jbang.jar
[jbang] Deleting file /Users/max/.jbang/bin/jbang
[jbang] Deleting file /Users/max/.jbang/bin/jbang.cmd
Cache: /Users/max/.jbang/cache
Config: /Users/max/.jbang
Repository:/Users/max/.m2/repository
0.68.0
❯ jbang versin
[jbang] [ERROR] Script or alias could not be found or read: 'versin'
[jbang] Run with --verbose for more details
❯ jbang version
0.69.1
```

--- a/src/main/java/dev/jbang/cli/App.java
+++ b/src/main/java/dev/jbang/cli/App.java
@@ -69,6 +69,7 @@ class AppInstall extends BaseCommand {
 					throw new IllegalArgumentException(
 							"It's not possible to install jbang with a different name");
 				}
+				Util.setFresh(true);// TODO: workaround as url cache is not honoring changed redirects
 				installed = installJbang(force);
 			} else {
 				if ("jbang".equals(name)) {

